roles/satellite-clone/tasks/restore.yml

```diff
@@ -7,23 +7,26 @@
 
 - name: Restore postgresql data
   command: tar --selinux --overwrite -xvf {{ backup_dir }}/pgsql_data.tar.gz -C /
-  when: not rhel_migration
+  when: not rhel_migration and pgsql_data.stat.exists
 
 - include: restore_postgresql_migration.yml
```

Comment:
this need update. see https://github.com/RedHatSatellite/satellite-clone/pull/119/files#r107929449